src/applications/personalization/profile/tests/e2e/direct-deposit/unsupported-signin-provider.cypress.spec.js

```diff
@@ -1,24 +1,11 @@
-import { PROFILE_PATHS } from '@@profile/constants';
 import { mockGETEndpoints } from '@@profile/tests/e2e/helpers';
 
-import { makeUserObject } from '~/applications/personalization/common/helpers';
```

Comment:
So there is a `makeUserObject` function already existing. It's not _that_ flexible. Eventually, I would want to create something like that in the `mocks` folder and get all of our test data into that folder so it's not in a few different places.